<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageDialog</class>
 <widget class="QDialog" name="ManageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1136</width>
    <height>775</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1120</width>
    <height>745</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Attendance Management</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>CocogooseProTrial Darkmode</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;CocogooseProTrial Darkmode&quot;;
color: rgb(21, 59, 160);
padding:10px;
</string>
     </property>
     <property name="text">
      <string>ATTENDANCES MANAGEMENT</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>7</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>19</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: 63 10pt &quot;Montserrat SemiBold&quot;;</string>
         </property>
         <property name="text">
          <string>Dates:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="styleSheet">
          <string notr="true">font: 57 8pt &quot;Montserrat Medium&quot;;
background-color: rgb(249, 252, 255);</string>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>549</width>
            <height>592</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="dateButtonLayout"/>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_selected_date">
           <property name="styleSheet">
            <string notr="true">font: 63 10pt &quot;Montserrat SemiBold&quot;;</string>
           </property>
           <property name="text">
            <string>Date:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonShowAll">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">
QPushButton {
 /* Màu nền mặc định */
	background-color: rgb(105, 248, 255);
    border-radius: 5px;
	color: rgb(9, 28, 84);
	font: 8pt &quot;CocogooseProTrial Semilight&quot;;

}

QPushButton:hover {
	background-color: rgb(137, 227, 254);
}

QPushButton:pressed {
background-color: rgb(105, 248, 255);
}


</string>
           </property>
           <property name="text">
            <string>Show All Students</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonImport">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">
QPushButton {
 /* Màu nền mặc định */
	background-color: rgb(105, 248, 255);
    border-radius: 5px;
	color: rgb(9, 28, 84);
	font: 8pt &quot;CocogooseProTrial Semilight&quot;;

}

QPushButton:hover {
	background-color: rgb(137, 227, 254);
}

QPushButton:pressed {
background-color: rgb(105, 248, 255);
}


</string>
           </property>
           <property name="text">
            <string>Import Class List</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonStatistic">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">
QPushButton {
 /* Màu nền mặc định */
	background-color: rgb(105, 248, 255);
    border-radius: 5px;
	color: rgb(9, 28, 84);
	font: 8pt &quot;CocogooseProTrial Semilight&quot;;

}

QPushButton:hover {
	background-color: rgb(137, 227, 254);
}

QPushButton:pressed {
background-color: rgb(105, 248, 255);
}


</string>
           </property>
           <property name="text">
            <string>Statistics</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(249, 252, 255);
color: rgb(14, 23, 67);
font: 57 10pt &quot;Montserrat Medium&quot;;
</string>
         </property>
         <column>
          <property name="text">
           <string>Student ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Full Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Time</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Status</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Image Registed</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton {
 /* Màu nền mặc định */
	background-color: qlineargradient(spread:pad, x1:0.134328, y1:0.182, x2:1, y2:0, stop:0 rgba(12, 33, 158, 255), stop:1 rgba(26, 72, 255, 255));
	color: rgb(255, 255, 255);
    border-radius: 5px;
	font: 57 10pt &quot;Montserrat Medium&quot;;

}

QPushButton:hover {
	
	background-color: rgb(29, 60, 199);

}

QPushButton:pressed {
    background-color: #0581F9; /* Màu khi nhấn */
}

</string>
       </property>
       <property name="text">
        <string>     Close     </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
